metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "puppetlabs-selinux_core",
   "version": "1.0.1",
-  "author": "Puppet Labs",
+  "author": "puppetlabs",
```

Comment:
As on Cron, should this be "Puppet" or something  "human readable" like that, raher than just matching our github namespace?

Reply:
I'm +1 to "Puppet"

Reply:
I think @binford2k  mentioned in another commit that this has to correspond to the forge user that publishes the module, so it should remain `puppetlabs`. I'm not seeing where that PR went though

Reply:
Ok, sounds good to me, thanks @melissa